docs/UserGuide.adoc

```diff
@@ -290,6 +290,8 @@ The student shares notes and questions with other students with Internet connect
 === Login using email and password `[coming in v2.0]`
 The student can register and login to the application using their email and password.
 
+=== Flash card grouping suggestions `[coming in v2.0]`
```

Comment:
This feature has the potential to differentiate our app more from other revision apps. But the actual implementation would be more complex.